docs/units/equivalencies.rst

```diff
@@ -347,6 +347,20 @@ and you want to know how big your pixels need to be to cover half an arcsecond::
     >>> (0.5*u.arcsec).to(u.micron, tel_platescale)  # doctest: +FLOAT_CMP
     <Quantity 18.9077335632719 micron>
 
+Photometric Zero Point Equivalencies
```

Comment:
What about converting to `STmag`? That should require some kind of wavelength as well like `spectral_density`?

Reply:
I don't quite understand what you mean here @pllim - in principle `STMag` (really any mag) always require a filter, which means the wavelength is implicitly set by the filter, right? That is, there is only one "valid" wavelength for a given something-band magnitude?


(regardless, should clarify in the docs, but wanted to make sure I'm not missing something first)

Reply:
I guess the example could be expanded to convert to STMag, use `spectral_density` to move to a different wavelength, and then back again?  But maybe that's overkill?

Reply:
Overkill, maybe not, although might be out of scope if you think it involves an actual bandpass.